ftcosi/protocol/protocol.go

```diff
@@ -347,7 +349,9 @@ func (p *FtCosi) startSubProtocol(tree *onet.Tree) (*SubFtCosi, error) {
 	cosiSubProtocol.Publics = p.publics
 	cosiSubProtocol.Msg = p.Msg
 	cosiSubProtocol.Data = p.Data
-	cosiSubProtocol.Timeout = p.Timeout / 2
+	// We allow for one subleader failure during the commit phase, and thus
```

Comment:
Why the choice to allow only one subleader failure?
Moreover, if the timeout for the commit phase is t/2 and for one subleader failure t/3, the second subleader will only have t/6 to complete before commit timeout is reached. Am I correct? @ineiti